scenes/quests/story_quests/stella/3_stella_sequence_puzzle/stella_sequence_puzzle.tscn

```diff
@@ -269,42 +269,51 @@ stream = ExtResource("20_koksn")
 y_sort_enabled = true
 
 [node name="HintSign1" parent="OnTheGround/SequencePuzzle/Signs" instance=ExtResource("11_c8urv")]
-position = Vector2(417, 314)
+position = Vector2(571, 417)
 sprite_frames = ExtResource("22_koksn")
 
 [node name="HintSign2" parent="OnTheGround/SequencePuzzle/Signs" instance=ExtResource("11_c8urv")]
-position = Vector2(609, 332)
+position = Vector2(703, 353)
 sprite_frames = ExtResource("23_g4uwj")
 
+[node name="HintSign3" parent="OnTheGround/SequencePuzzle/Signs" instance=ExtResource("11_c8urv")]
+position = Vector2(830, 279)
+sprite_frames = ExtResource("24_7w21t")
+
 [node name="Steps" type="Node2D" parent="OnTheGround/SequencePuzzle"]
 
 [node name="SequencePuzzleStep1" type="Node2D" parent="OnTheGround/SequencePuzzle/Steps" node_paths=PackedStringArray("sequence", "hint_sign")]
 script = ExtResource("13_cyoxr")
-sequence = [NodePath("../../Objects/Gem 2"), NodePath("../../Objects/Gem 4"), NodePath("../../Objects/Gem 1")]
+sequence = [NodePath("../../Objects/Gem 2"), NodePath("../../Objects/Gem 4"), NodePath("../../Objects/Gem 6")]
 hint_sign = NodePath("../../Signs/HintSign1")
 
 [node name="SequencePuzzleStep2" type="Node2D" parent="OnTheGround/SequencePuzzle/Steps" node_paths=PackedStringArray("sequence", "hint_sign")]
 script = ExtResource("13_cyoxr")
 sequence = [NodePath("../../Objects/Gem 2"), NodePath("../../Objects/Gem 4"), NodePath("../../Objects/Gem 6"), NodePath("../../Objects/Gem 5"), NodePath("../../Objects/Gem 3"), NodePath("../../Objects/Gem 1")]
 hint_sign = NodePath("../../Signs/HintSign2")
 
+[node name="SequencePuzzleStep3" type="Node2D" parent="OnTheGround/SequencePuzzle/Steps" node_paths=PackedStringArray("sequence", "hint_sign")]
```

Comment:
You haven't added this to the Steps list in the SequencePuzzle

![image](https://github.com/user-attachments/assets/8610885a-e542-4fba-9091-74ca7ac918e5)